src/asynk/async_queue.rs

```diff
@@ -76,13 +110,6 @@ where
             return Err(AsyncQueueError::PoolAndTransactionEmpty);
         };
 
-        if result != 1 {
```

Comment:
we can rename `execute_one` into `execute` and add additional parameter - expected_result_count: Option<u64>

```rust
if let Some(expected_result) = expected_result_count {
  if result != expected_result {
    return Err(AsyncQueueError::ResultError {
                expected: expected_result,
                found: result,
            });
  }
}
```

Reply:
@ayrat555 ,  Ok i will add this code, but we should call the function different because there is already an execute function Transaction::execute. Maybe will be better call  it ```execute_test``` because we are gonna to pass an expected_result.

Reply:
I think it's fine to call it `execute`. because we're adding the method to our own struct

but you can call it `checked_execute` if you don't like just `execute`

Reply:
@ayrat555 is a problem to add this parameter to expected_result_count: Option to all functions ?

Reply:
I think it's fine. but the code will be duplicated

Reply:
@ayrat555 if we change all tests and only call Checked execute in the tests we can avoid the duplicated code.

Reply:
@ayrat555 why would you like this error in checked_execute ? 
asser_eq! macro already gives the expected value and the value given.

![image](https://user-images.githubusercontent.com/48651252/179470486-cfb587c6-0e16-4298-893a-110872649cda.png)
I temporary modified the code to show this example.